lib/identity_pki/certificate.ex

```diff
@@ -38,6 +38,23 @@ defmodule IdentityPki.Certificate do
     end
   end
 
+  @doc """
+  SHA512 digests the RFC 2253 formatted subject of a certificate.
+
+  The digest is Base64 encoded, but a `"\n"` is added every 60 characters
+  to match the behavior of Ruby's `Base64.encode64`
```

Comment:
This was a learning experience: https://stackoverflow.com/a/2621023

Reply:
Oof that sounds like fun